modules/network-observability-flow-filter-parameters.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: REFERENCE
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+[id="network-observability-flowcollector-flowfilter-parameters_{context}"]
+= Flow filter configuration parameters
+The flow filter rules consist of mandatory and optional parameters.
+
+.Required configuration parameters
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`enable`
+| Set `enable` to `true` to enable eBPF flow filtering feature.
+
+|`cidr`
+| Provides the IP address and CIDR mask for the flow filter rule, supports IPv4 and IPv6 address format. If you want to match against any IP, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6 respectively.
+
+|`action`
+| Describes the action to be taken for the flow filter rule. The possible values are `Accept` or `Reject`.
+
+* For the matching rule with the `Accept` action, this flow is allowed to be cached in the eBPF table, with the updated global metric `FlowFilterAcceptCounter`.
+* For the matching rule with `Reject` action, this flow is dropped and not cached in eBPF table, with the updated global metric `FlowFilterRejectCounter`.
+* If the rule is not matched, the flow is allowed to be cached in eBPF table, with the updated global metric `FlowFilterNoMatchCounter`.
+|===
+
+
+.Optional configuration parameters
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`direction`
+| Defines the direction of the flow filter rule. Possible values are `Ingress` or `Egress`.
+
+|`protocol`
+| Defines the protocol of the flow filter rule. Possible values are `TCP`, `UDP`, `SCTP`, `ICMP`, `ICMPv6`.
+
+| `ports`
+| Defines the ports to filter flows by. It can be used for either source or destination ports. To filter a single port, set a single port as an integer value. For example `ports: 80`. To filter a range of ports, use a "start-end" range in string format. For example `ports: "80-10"`
+
+|`sourcePorts`
+| Defines the source port to filter flows by. To filter a single port, set a single port as an integer value, for example `sourcePorts: 80`. To filter a range of ports, use a "start-end" range, string format, for example `sourcePorts: "80-100"`.
+
+
+| `destPorts`
+| DestPorts defines the destination ports to filter flows by. To filter a single port, set a single port as an integer value, for example `destPorts: 80`. To filter a range of ports, use a "start-end" range in string format, for example `destPorts: "80-100"`.
+
+| `icmpType`
+| Defines the ICMP type to filter flows by.
+
+| `icmpCode`
+| Defines the ICMP code to filter flows by.
```

Comment:
```suggestion
| Defines the ICMP code to filter the flows.
```

Reply:
I updated these to say "to use for filtering flows"